docs/operation/account_administration.rst

```diff
@@ -0,0 +1,86 @@
+
+Account Administration
+======================
+
+This section highlights **MetalK8s Account Administration** which covers
+changing the default username and password for some MetalK8s services.
+
+
+Administering Grafana
+*********************
+
+A fresh install of MetalK8s has a Grafana service instance with default
+credentials: ``admin`` / ``admin``. For more information on how to access
+Grafana, please refer to :ref:`this procedure <quickstart-services-grafana>`
+
+Changing Grafana username and password
+--------------------------------------
+
+To change the default username and password for Grafana on a MetalK8s cluster,
+perform the following procedures:
+
+#. Generate a new Grafana username and password using a **Base64 encoder**
+   and note it down.
+
+   .. code-block:: shell
+
+      $ echo -n 'new-username' | base64
+      $ echo -n 'new-password' | base64
+
+#. Create a file named ``patch-secret.yaml`` that has following content:
+
+   .. code-block:: yaml
+
+      data:
+        admin-password: bmV3cGFzc3dvcmQ=   <-- Replace with base64 password
+        admin-user: bmV3cGFzc3dvcmQ=       <-- Replace with base64 username
+        ldap-toml: ""
```

Comment:
Let's remove the `base64` encoding for the user, since API Server supports doing it server-side. Use this for the patch file instead:

```
stringData:
  admin-user: <username-in-clear>
  admin-password: <password-in-clear>
```

(Note that we don't need to patch `ldap-toml`)

Also, could be nice mentioning that either one of the two keys can be omitted from this patch, depending on what the user wants to achieve.